src/display/svg.js

```diff
@@ -665,8 +681,8 @@ SVGGraphics = (function SVGGraphicsClosure() {
           case OPS.endPath:
             this.endPath();
             break;
-          case 92:
-            this.group(opTree[x].items);
+          case OPS.group:
+            this.createGroup(opTree[x].items);
```

Comment:
This can still be `group` if you rename the member variables as indicated in the comment above.